File: domain/src/test/java/com/ricamgar/notify/domain/reminder/RemindersDoubles.java
package com.ricamgar.notify.domain.reminder;

import com.ricamgar.notify.domain.reminder.model.Reminder;

import java.util.Collections;
import java.util.List;

public final class RemindersDoubles {
    public static final Reminder REMINDER = new Reminder.Builder()
            .id(0)
            .description("description")
            .sticky(true)
            .done(false)
            .build();
    public static final List<Reminder> REMINDERS_LIST = Collections.singletonList(REMINDER);
}

File: domain/src/test/java/com/ricamgar/notify/domain/reminder/RemindersDoubles.kt
package com.ricamgar.notify.domain.reminder

import com.ricamgar.notify.domain.reminder.model.Reminder

object RemindersDoubles {
    val REMINDER: Reminder = Reminder.Builder().id(0).description("description").done(false).build()
    val REMINDERS_LIST = listOf(REMINDER)
}
